test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -5970,7 +5970,7 @@ private static void VerifyNuspecRoundTrips(PackageArchiveReader nupkgReader, str
             }
         }
 
-        [Fact]
+        //[Fact]
```

Comment:
Do using `Fact(Skip = "...")` attribute would be better? Just not to forget to re-activate the tests when re-enabling deterministic pack

Reply:
We're not merging this back to dev, this is only a release change, so I just went for the simplest fix.